WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsSelectPluginView.swift

```diff
@@ -27,8 +27,11 @@ struct InPersonPaymentsSelectPluginRow: View {
                 .stroke(borderColor, lineWidth: 1)
                 .background(Color(.tertiarySystemBackground))
                )
+        .accessibilityElement(children: .combine)
+        .accessibilityLabel("\(name)." +
+                            "\(Localization.accessibilitySelectableString)." +
+                            "\(selected ? Localization.accessibilitySelectedString : Localization.accessibilityNotSelectedString)")
```

Comment:
```suggestion
        .accessibilityElement(children: .combine)
        .accessibilityLabel(name)
        .accessibilityRemoveTraits([.isImage])
        .accessibilityAddTraits([.isButton])
        .accessibilityAddTraits(selected ? [.isSelected] : [])
```

Your solution works well, but it does combine strings in a way which might not localize well in all languages.

Apple provides API for these by adding and removing traits, which it then localizes for us, and will match the VoiceOver experience for other controls, and also translate better to other non-VoiceOver accessibility features, e.g. Switch Control, braille readers, etc.

Reply:
Here's what it sounds like:
https://user-images.githubusercontent.com/2472348/176691260-fd71c122-abe2-4806-9813-7729b169a6b1.mp4